client/my-sites/domains/domain-management/transfer/transfer-to-other-site/confirmation-dialog.jsx

```diff
@@ -76,7 +83,7 @@ class TransferConfirmationDialog extends PureComponent {
 		const { isMapping, translate } = this.props;
 		const actionLabel = ! isMapping
 			? translate( 'Confirm Transfer' )
-			: translate( 'Confirm Mapping Transfer' );
+			: translate( 'Confirm Domain Connection Transfer' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Transfer this domain connection' )` *ES Score: 7*